samples/templates/webpack-spa/webpack.config.js

```diff
@@ -36,15 +36,15 @@ module.exports = {
     publicPath: '/'
   },
   devServer: {
-    contentBase: [
+    static: [
       path.join(__dirname, 'public'),
       SIW_DIR
     ],
     port: PORT,
-    before: function(app) {
-      app.get('{{ redirectPath }}', redirectToOrigin);
-      app.get('/login', redirectToOrigin);
-      app.get('/profile', redirectToOrigin);
+    onBeforeSetupMiddleware: function(devServer) {
+      devServer.app.get('/login/callback', redirectToOrigin);
```

Comment:
is there a reason we can't support the template here?

Reply:
Not sure I understand the question. This syntax changed due to a major version bump.

https://github.com/okta/okta-auth-js/pull/1218/files#diff-25616d4e5ff9af120aa428e51aeaaa20505440f3e3945373607ba8b72cd35292R26

https://github.com/webpack/webpack-dev-server/blob/master/CHANGELOG.md#-breaking-changes-4

> before, after and setup were removed in favor onBeforeSetupMiddleware (previously before) and onAfterSetupMiddleware options (previously after)

Reply:
nevermind, fixed

Reply:
I mean the template `{{ redirectPath }}`

Reply:
Yea, this was fixed: https://github.com/okta/okta-auth-js/pull/1218/files#diff-517e04b9fefa77f6361bc25d43ae5ce8af3cd841caa15f73b3ac5eca247193e9R45